src/client/src/orangehrmClaimPlugin/pages/submitClaim/SubmitClaim.vue

```diff
@@ -40,7 +40,7 @@
               </oxd-grid-item>
               <oxd-grid-item>
                 <oxd-input-field
-                  v-model="claimEvent.name"
+                  v-model="formattedEventName"
```

Comment:
@Super-Chama can't we use placeholder here ?

Reply:
can, what's the usecase here? claim event name should be pre populated?